content/1_General/Resources_CP.mdx

```diff
@@ -25,9 +25,10 @@ These resources are frequently referenced within this guide.
 - Intro to USACO (Bronze, Silver) - Darren Yao
   - [Java](http://darrenyao.com/usacobook/java.pdf), [C++](http://darrenyao.com/usacobook/cpp.pdf) versions
 - [Principles of Algorithmic Problem Solving](http://www.csc.kth.se/~jsannemo/slask/main.pdf) - Johan Sannemo
-  - practice problems from Kattis
+  - practice problems from Kattis.
 - Competitive Programming Book - Steven Halim, Felix Halim
-  - [Competitive Programming 2](https://www.comp.nus.edu.sg/~stevenha/myteaching/competitive_programming/cp2.pdf) is freely available but old
+  - [Competitive Programming 2](https://www.comp.nus.edu.sg/~stevenha/myteaching/competitive_programming/cp2.pdf) is freely available but old.
+  - [Competitive Programming 3](https://github.com/prasadgujar/CompetitiveProgramming/blob/master/book/Competitive%20Programming%203.pdf) is free and relevant.
```

Comment:
CP3 is technically not free. I'm unsure whether we should include a link to a book that's still available for sale. Maybe at most just subtly hint that it can be found online (while also including a link to the official website or something idk).

Reply:
wdyt of "is relevant and can be found online" while linking to their official?

Reply:
sure?

Reply:
That kinda sounds like you're encouraging piracy though?

Reply:
difficult decisions :c

(maybe "can be found online" is too strong and we should just be like "apparently the book can also be found online.")

(Or we should just assume that people are smart enough to know how to use google and not mention this at all ://

Reply:
Hmmm I guess we should probably we should assume that people are smart enough, since many of these topics are covered anyways. Also I feel "apparently" draws more emphasis towards to it so maybe not the best wording.

Reply:
Add "monetized" or "paid" resources section? (At which point I think youtube technically falls under that category, but it's a bit different)

I guess it comes down to whether we want to promote monetized resources.

Reply:
I'm personally fine with promoting monetized resources (as long as the core team agrees that the resource is good)?

Reply:
Yea, I agree. TBH if you said you weren't fine I would have agreed too. So in other words I don't really have an opinion